src/GRA.Controllers/MissionControl/TriggersController.cs

```diff
@@ -361,6 +383,10 @@ await _siteService.GetBranches(model.Trigger.LimitToSystemId.Value),
             {
                 model.MaxPointsMessage = $"(Up to {model.MaxPointLimit.Value} points)";
             }
+            if (model.MinAllowedPoints.HasValue)
+            {
+                model.MinPointsMessage = $"(Must have at least {model.MinAllowedPoints.Value} points)";
```

Comment:
This text also could be in the view if what was passed into the view model was an `int?` with the value.